spec/parser/options_spec.rb

```diff
@@ -116,7 +116,9 @@ def remaining
       expected = "Unknown switches \"--baz\""
       expected << "\nDid you mean?  \"--bar\"" if Thor::Correctable
 
-      expect { check_unknown! }.to raise_error(Thor::UnknownArgumentError, expected)
```

Comment:
I wonder, shouldn't be rather the `Thor::Correctable` line above dropped? But that would probably not be backward compatible with older rspec-mock :thinking:

This is caused by https://github.com/rspec/rspec-expectations/pull/1339 just for the context.

Reply:
@voxik sorry for confusion, this comes from #780, because I wanted this to be green first before proposing a change.

Can we continue the discussion in #780 ?

Reply:
https://github.com/rails/thor/pull/780/files#r823694117

Reply:
Ah, sorry, right. I have noticed just now you opened multiple PRs and there is one for this case specifically.

Reply:
Actually this comes from #777 :)